### PR TITLE
Updated Main Screen 

### PR DESCRIPTION
**Pull request by akshaydhiran**
Final Main Screen 
@TahaAli07 , please review our pull request.
UpdatedMainScreenDesign

### CODE REVIEW

**Comment by TahaAli07** [comment]
@akshaysingh5 You cam improvise the UI by reducing the size of the icons and adding text below it - Share Location and Send Alert

**Comment by akshaydhiran** [comment]
Okay

**Review by tanmayrauth — COMMENTED** [review]
Can't we use material io designs for UI ? This UI is looking to me a bygone  one.

**Review by akshaydhiran — COMMENTED** [review]
Updated Main Screen Design 

MainScreenLatestDesign


Collaborative work of @akshaysingh5 , @Ruchika30 and @sheen404

**Comment by akshaydhiran** [comment]
Please review @TahaAli07 , @ankitshaw .

**Comment by TahaAli07** [comment]
@akshaysingh5 the bottom navigation bar looks out of place , the icons look unbalanced. Can you fix that?

**Comment by akshaydhiran** [comment]
Earlier it was approved by @ankitshaw and he suggests some changes and we did it, that's why i update this PR , @TahaAli07

**Comment by TahaAli07** [comment]
Ok no problem, we will look at it later . Good work !

**Comment by ankitshaw** [comment]
@akshaysingh5  can you use the color scheme used in the logo? and bottom bar is too high

**Comment by akshaydhiran** [comment]
Okay and color scheme for what @ankitshaw

**Comment by ankitshaw** [comment]
@akshaysingh5  change orange and grey to the blue shade in the logo

**Review by akshaydhiran — COMMENTED** [review]
Final Main Screen

mainscreenupdated

**Comment by akshaydhiran** [comment]
@ankitshaw and @TahaAli07 , Kindly Check.

**Comment by ankitshaw** [comment]
@akshaysingh5  as I said earlier reduce botton padding for the navigation bar

**Review by akshaydhiran — COMMENTED** [review]
Done @ankitshaw .

LastUpdatedMainScreen

**Comment by ankitshaw** [comment]
@akshaysingh5 merge conflicts I can see

**Comment by akshaydhiran** [comment]
This is because of someone's merge approved before us.

File: master/lib/main.dart [hunk 1]
```diff
@@ -1,163 +1,19 @@
 import 'package:flutter/material.dart';
-import 'package:splashscreen/splashscreen.dart';
```
Comment (ankitshaw) [inline master/lib/main.dart]: removed spash screen login screen ????

File: master/lib/main.dart [hunk 2]
```diff
@@ -1,163 +1,19 @@
...
 
-class _MyAppState extends State<MyApp> {
+class MyApp extends StatelessWidget{
   @override
-  Widget build(BuildContext context) {
-    return new SplashScreen(
-      seconds: 14,
-      navigateAfterSeconds: new MyHomePage(),
-      title: new Text(
-        'Welcome To Travel Safe',
```
Comment (ankitshaw) [inline master/lib/main.dart]: removed?

**Review by akshaydhiran — COMMENTED** [review]
It was deleted by mistake earlier but everything is fine now @ankitshaw

**Pull request closed without merging**